client/src/App.css

```diff
@@ -10,3 +10,46 @@ body{
   left: 50%;
   transform: translate(-50%, -50%);
 }
+
+.App {
+  text-align: center;
+}
+
+.container {
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.App-logo {
```

Comment:
Are we using this class anymore?
Please remove it if not used.

Reply:
Removed